sdk/storage/azblob/blob/client_test.go

```diff
@@ -191,6 +191,67 @@ func waitForCopy(_require *require.Assertions, copyBlobClient *blockblob.Client,
 	}
 }
 
+func (s *BlobUnrecordedTestsSuite) TestCopyBlockBlobFromURL() {
```

Comment:
Is this a live-only test, or where is the test recording?

Reply:
CopyFromURL uses SAS to authenticate and currently there is no way to record SAS related tests.

Reply:
Makes sense.  Do we have an issue open with Azure SDK to enable us to record SAS tests?